src/roiextractors/extractors/hdf5imagingextractor/hdf5imagingextractor.py

```diff
@@ -108,7 +108,7 @@ def get_frames(self, frame_idxs, channel=0):
             # return lazy_ops.DatasetView(self._video).lazy_slice[channel, sorted_idxs].lazy_slice[:, argsorted_idxs]
 
     def get_image_size(self):
```

Comment:
Maybe add annotation typing returns to super clarify this, also all the way up to the abstract level? 😁

```python
def get_image_size(self) -> Tuple[int, int]:
```

Reply:
Otherwise yeah, totally supportive of this. Thanks for the small quick PR!

Reply:
Should be done now.